### PR TITLE
explicitly specify nodejs version

### PR DESCRIPTION
**Pull request by cgardens**
## What
* In our docs we say that we use nodejs 14. 

## Current behavior
* We use whatever version of nodejs that apt-get gives us (which right now just happens to be node14

## Desired behavior (after this PR)
* It should be easy to identify what version of nodejs we actually pull in. After Artem asked about what version we were using, it took too long for me to figure out the actual answer.
* We are explicit about what version of nodejs we are using with apt-get. If our desired version ceases to be available, at least we will know and be able to take action as opposed to getting magically upgraded

### CODE REVIEW

**Review by jrhizor — APPROVED** [review]

File: tools/build/Dockerfile.base [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN apt-get update && apt-get -y install curl
 # Setup Node & Java
 RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
 RUN apt-get update && apt-get -y install \
-  nodejs \
+  nodejs=14.10.1-deb-1nodesource1 \
```
Comment (michel-tricot) [inline tools/build/Dockerfile.base:16]: we actually don't use that image anymore now that we moved the dockerfiles into their respective projects

File: tools/build/Dockerfile.base [hunk 2]
```diff
@@ -13,7 +13,7 @@ RUN apt-get update && apt-get -y install curl
 # Setup Node & Java
 RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
```
Comment (michel-tricot) [inline tools/build/Dockerfile.base:14]: doesn't that take care of the version?

File: tools/build/Dockerfile.base (hunk 1 above)
Comment (cgardens) [reply tools/build/Dockerfile.base:16]: oh. can we remove it entirely?

Comment (michel-tricot) [reply tools/build/Dockerfile.base:16]: yes we can

**Comment by cgardens** [comment]
In response to feedback...
* removed dataline/build-base (unused)
* set nodejs version in the gradle plugin, so that we always use the right version as opposed to whatever happens to be present on the local machine.

**Pull request merged**